ccdaservice/serveccda.js

```diff
@@ -1201,8 +1223,8 @@ function getResultSet(results) {
             {
                 "date_time": {
                     "point": {
-                        "date": fDate(tResult.date_ordered),
-                        "precision": getPrecision(fDate(tResult.date_ordered))
+                        "date": fDate(authorDateTime),
+                        "precision": "tz"
```

Comment:
at this point authorDateTime has already been through fDate(). Won't hurt though to ensure in case global formatting is removed in future.